assets/styles/custom/medialib.scss

```diff
@@ -149,6 +156,22 @@ $img-border-strength: 2px;
         }
       }
     }
+    & .header {
+      position: relative;
+      font-size: 1.5em;
+      cursor: pointer;
+    }
+    & .header::before {
+      content: '\25BC';
```

Comment:
maybe you could check with someone from the UX team, if we should use material icons here for consistency. (Can be done in a new PR, if necessary)